front/app/components/admin/FormResults/FormResultsPage/index.tsx

```diff
@@ -13,7 +13,7 @@ import { LogicConfig, ResultUngrouped } from 'api/survey_results/types';
 
 import useLocalize from 'hooks/useLocalize';
```

Comment:
NOTE: Most of the changed files in this PR are because I moved FormResults into the shared components/admin folder (since we're re-using it in the Community Monitor now).